components/Common/Sidebar/SidebarGroup/index.module.css

```
@@ -0,0 +1,24 @@
+.group {
+  @apply flex
+  w-full
+  flex-col
+  gap-2;
+}
```

```suggestion
.group {
  @apply flex
    w-full
    flex-col
    gap-2;
}
```

Missing indentations